src/uwtools/apps/srw.py

```diff
@@ -7,7 +7,7 @@
 from uwtools.drivers.facade import Facade
 from uwtools.exceptions import UWError
 from uwtools.logging import log
-from uwtools.utils.file import FORMAT, get_file_type
+from uwtools.utils.file import FORMAT, get_file_format
```

Comment:
We use `format` pretty pervasively to describe YAML, namelist, INI, etc., so use that instead of `type`.